src/main/java/gregtech/common/metatileentities/multi/electric/generator/LargeTurbineWorkableHandler.java

```diff
@@ -44,6 +54,9 @@ public FluidStack getInputFluidStack() {
         if (previousRecipe == null) {
             Recipe recipe = findRecipe(Integer.MAX_VALUE, getInputInventory(), getInputTank());
 
+            // If there is no runnable recipe, attempt to show a candidate recipe fluid stack
+            recipe = recipe != null ? recipe : super.findRecipe(Integer.MAX_VALUE, getInputInventory(), getInputTank());
+
             return recipe == null ? null : getInputTank().drain(
                     new FluidStack(recipe.getFluidInputs().get(0).getInputFluidStack().getFluid(), Integer.MAX_VALUE),
                     false);
```

Comment:
the issue with this is that `findRecipe()` will always return null over `super.findRecipe()`, since there's won't be a runnable recipe. then, once there *is* a runnable recipe, `previousRecipe` is not null and the logic gets skipped

Reply:
Changed in cef1e27ca